_15thnight/core.py

```diff
@@ -1,10 +1,17 @@
+from flask import render_template, url_for
 from flask.ext.login import current_user
+from flask_mail import Message
 
-from _15thnight.queue import queue_send_message
+from _15thnight.queue import queue_send_message, queue_send_email
 from _15thnight.models import (
     Alert, Need, NeedProvided, ProviderNotified, Service, Response, User
 )
 
+try:
+    from config import SUPPORT_EMAIL
```

Comment:
If possible, it'd be better if the SUPPORT_EMAIL value were grabbed from the app's config.

```python
from flask import current_app

support_email = current_app.config.get("SUPPORT_EMAIL")